### ISSUE TITLE
Accessibility issue with Accordion keyboard navigation

### ISSUE THREAD

**Issue by ltaylor2607** [primefaces/primevue#1616]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primevue/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=110
```

**CodeSandbox Case (Bug Reports)**
https://codesandbox.io/s/accordion-accessibility-issue-wcxxi?file=/src/App.vue

**Current behavior**
When using the keyboard to navigate through the accordion, it is not possible to open the accordion tab using the enter key.

**Expected behavior**
Pressing the enter key on an accordion tab should open the tab and display the content.

**Minimal reproduction of the problem with instructions**
A simple reproduction can be found on CodeSandbox at https://codesandbox.io/s/accordion-accessibility-issue-wcxxi?file=/src/App.vue
Navigate through the accordion using the keyboard and attempt to use enter to open the accordion tab

**What is the motivation / use case for changing the behavior?**
For accessibility purposes, it should be possible to navigate through an accordion using only the keyboard.

**Please tell us about your environment:**
Operating system: Windows 10 Business
IDE: Visual Studio Code
Package manager: npm
Server: server provided by vue-cli-server

* **Vue version:** 3.X
3.1.2

* **PrimeVue version:** 3.5.X
3.5.1

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
Microsoft edge version 94.0.992.31
Firefox version 91.0.2
Chrome version 94.0.4606.61
